src/oop/programs_with_multiple_classes/hard/q3/Bookcase.java

```diff
@@ -0,0 +1,9 @@
+package oop.programs_with_multiple_classes.hard.q3;
+
+public class Bookcase {
+    private int sizeCapacity;
+
+    public Bookcase(int sizeCapacity) {
+        this.sizeCapacity = sizeCapacity;
```

Comment:
Also, can we add a check for this constructor to make sure the sizeCapacity is not negative?